_episodes/07-conclusions.md

```diff
@@ -38,101 +50,45 @@ From here, there are lots of ways that you can continue to use ESMValTool.
 >   - Is any observational data available?
 >   - How will you preprocess the data?
 >   - What will your diagnostic script need to do?
->   - What will your final figure show? 
+>   - What will your final figure show?
 {: .challenge}
 
 ### Where can I get more information on ESMValTool?
 
-> ## Additional resources: 
+> ## Additional resources:
+>
 > - [Read the docs page](https://esmvaltool.readthedocs.io/)
 > - [ESMValTool home page](https://www.esmvaltool.org/)
-> - [Technical description paper](https://doi.org/10.5194/gmd-13-1179-2020)
+> - [Technical description papers](https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/master/CITATION)
```

Comment:
```suggestion
> - [Papers](https://github.com/ESMValGroup/ESMValTool_Tutorial/blob/master/CITATION)
```
there is only 1 technical paper, the others describe the scientific analyses made available through the included recipes